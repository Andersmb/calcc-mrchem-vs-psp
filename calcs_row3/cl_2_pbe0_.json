{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            0.0,
            0.0,
            -0.06658603330372151
          ],
          "radius": 0.97
        },
        {
          "center": [
            0.0,
            0.0,
            2.3420587166962785
          ],
          "radius": 0.79
        }
      ],
      "cavity_width": 0.2,
      "charge": 0,
      "coords": [
        {
          "atom": "cl",
          "xyz": [
            0.0,
            0.0,
            -0.06658603330372151
          ]
        },
        {
          "atom": "h",
          "xyz": [
            0.0,
            0.0,
            2.3420587166962785
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": 1,
      "numerically_exact": false,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 9,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "cl_2_pbe0_.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-06,
          "shared_memory": true
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-06
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-06,
          "shared_memory": true,
          "smooth_prec": 1e-06
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe0"
              }
            ],
            "spin": true
          }
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": true,
        "method": "DFT (PBE0)",
        "prec": 0.0001,
        "relativity": "None",
        "restricted": false,
        "screen": 12.0,
        "type": "sad_gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-06,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "derivative": "abgv_55",
        "energy_thrs": 1e-06,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 1e-06,
        "helmholtz_prec": -1.0,
        "kain": 10,
        "light_speed": -1.0,
        "localize": true,
        "max_iter": 200,
        "method": "DFT (PBE0)",
        "orbital_thrs": 4.9999999999999996e-05,
        "proj_prec": 1e-06,
        "relativity": "None",
        "rotation": 0,
        "shared_memory": true,
        "smooth_prec": 1e-06,
        "start_prec": 1e-06
      },
      "write_orbitals": {
        "file_phi_a": "orbitals/phi_a_scf",
        "file_phi_b": "orbitals/phi_b_scf",
        "file_phi_p": "orbitals/phi_p_scf"
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        -9.268114993664536e-18
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.4369398268472839,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            3.6073961380665035e-07,
            2.8691078351760985e-06,
            0.43693982683771515
          ],
          "vector_el": [
            3.606393035315847e-07,
            2.8690076820346577e-06,
            -0.7731563236025705
          ],
          "vector_nuc": [
            1.0031027506565132e-10,
            1.0015314144093759e-10,
            1.2100961504402856
          ]
        }
      },
      "geometry": [
        {
          "symbol": "Cl",
          "xyz": [
            0.0,
            0.0,
            -0.06658603330372151
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            2.3420587166962785
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -101.71513646495858,
          -9.541618128592852,
          -7.275514408553892,
          -7.267725769942339,
          -7.267725632166887,
          -0.8702351413848536,
          -0.4894985642901735,
          -0.3493096670709277,
          -0.34930951113037073,
          -101.71513646525503,
          -9.541618128912262,
          -7.275514409679388,
          -7.267725770132145,
          -7.267725632352624,
          -0.8702351388392221,
          -0.4894985520991284,
          -0.34930966778250105,
          -0.34930951183927944
        ],
        "occupation": [
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0
        ],
        "spin": [
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b"
        ],
        "sum_occupied": -270.25214656498247
      },
      "scf_energy": {
        "E_ee": 210.75398123404605,
        "E_eext": 0.0,
        "E_el": -467.7387257179756,
        "E_en": -1110.138766498087,
        "E_kin": 460.09358037220875,
        "E_next": 0.0,
        "E_nn": 7.057910885364062,
        "E_nuc": 7.057910885364062,
        "E_tot": -460.6808148326116,
        "E_x": -6.96816714547436,
        "E_xc": -21.479353680668982,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 30,
      "nthreads": 6,
      "routine": "mrchem.x",
      "total_cores": 180,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 211.56112783561707,
        "E_eext": 0.0,
        "E_el": -465.4622392198372,
        "E_en": -1106.3330628767385,
        "E_kin": 457.76755279261687,
        "E_next": 0.0,
        "E_nn": 7.057910885364062,
        "E_nuc": 7.057910885364062,
        "E_tot": -458.4043283344732,
        "E_x": -6.975442045055423,
        "E_xc": -21.48241492627726,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 210.1261398742968,
              "E_eext": 0.0,
              "E_el": -467.66010662241723,
              "E_en": -1105.4693833361785,
              "E_kin": 456.04224644785205,
              "E_next": 0.0,
              "E_nn": 7.057910885364062,
              "E_nuc": 7.057910885364062,
              "E_tot": -460.6021957370532,
              "E_x": -6.949195361626691,
              "E_xc": -21.409914246760863,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -460.6021957370532,
            "energy_update": 2.197867402580016,
            "mo_residual": 0.18227600184829504,
            "wall_time": 87.146489744
          },
          {
            "energy_terms": {
              "E_ee": 210.6540363689356,
              "E_eext": 0.0,
              "E_el": -467.7258732258154,
              "E_en": -1108.282585435281,
              "E_kin": 458.31982321988824,
              "E_next": 0.0,
              "E_nn": 7.057910885364062,
              "E_nuc": 7.057910885364062,
              "E_tot": -460.66796234045137,
              "E_x": -6.961815025971403,
              "E_xc": -21.455332353386975,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -460.66796234045137,
            "energy_update": 0.06576660339817408,
            "mo_residual": 0.06339440234801373,
            "wall_time": 65.161166042
          },
          {
            "energy_terms": {
              "E_ee": 210.61135047584153,
              "E_eext": 0.0,
              "E_el": -467.73519273620457,
              "E_en": -1109.164002418521,
              "E_kin": 459.24312260647804,
              "E_next": 0.0,
              "E_nn": 7.057910885364062,
              "E_nuc": 7.057910885364062,
              "E_tot": -460.67728185084053,
              "E_x": -6.9632217889174415,
              "E_xc": -21.462441611085588,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -460.67728185084053,
            "energy_update": 0.009319510389161678,
            "mo_residual": 0.02524696934858709,
            "wall_time": 52.872265986
          },
          {
            "energy_terms": {
              "E_ee": 210.71461430225384,
              "E_eext": 0.0,
              "E_el": -467.73745386312964,
              "E_en": -1109.7480895237925,
              "E_kin": 459.7349493975829,
              "E_next": 0.0,
              "E_nn": 7.057910885364062,
              "E_nuc": 7.057910885364062,
              "E_tot": -460.6795429777656,
              "E_x": -6.96618248262958,
              "E_xc": -21.472745556544318,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -460.6795429777656,
            "energy_update": 0.0022611269250774058,
            "mo_residual": 0.015874029412583715,
            "wall_time": 57.252656363
          },
          {
            "energy_terms": {
              "E_ee": 210.76477365353367,
              "E_eext": 0.0,
              "E_el": -467.7384974648001,
              "E_en": -1110.3040309418225,
              "E_kin": 460.25020895354726,
              "E_next": 0.0,
              "E_nn": 7.057910885364062,
              "E_nuc": 7.057910885364062,
              "E_tot": -460.68058657943607,
              "E_x": -6.96847613099615,
              "E_xc": -21.480972999062445,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -460.68058657943607,
            "energy_update": 0.001043601670460248,
            "mo_residual": 0.015546662838702046,
            "wall_time": 56.245429276
          },
          {
            "energy_terms": {
              "E_ee": 210.76218392229552,
              "E_eext": 0.0,
              "E_el": -467.73865155185246,
              "E_en": -1110.240902939517,
              "E_kin": 460.18900224187803,
              "E_next": 0.0,
              "E_nn": 7.057910885364062,
              "E_nuc": 7.057910885364062,
              "E_tot": -460.6807406664884,
              "E_x": -6.968392195076828,
              "E_xc": -21.480542581432324,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -460.6807406664884,
            "energy_update": 0.00015408705235131492,
            "mo_residual": 0.006732048166993549,
            "wall_time": 56.629353304
          },
          {
            "energy_terms": {
              "E_ee": 210.76380179164636,
              "E_eext": 0.0,
              "E_el": -467.7387110329139,
              "E_en": -1110.1598307403692,
              "E_kin": 460.1057179323465,
              "E_next": 0.0,
              "E_nn": 7.057910885364062,
              "E_nuc": 7.057910885364062,
              "E_tot": -460.68080014754986,
              "E_x": -6.968359626220427,
              "E_xc": -21.48004039031721,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -460.68080014754986,
            "energy_update": 5.9481061441601923e-05,
            "mo_residual": 0.004785241973496013,
            "wall_time": 56.622493122
          },
          {
            "energy_terms": {
              "E_ee": 210.7579345101683,
              "E_eext": 0.0,
              "E_el": -467.73872094118775,
              "E_en": -1110.1402218610303,
              "E_kin": 460.09132265494986,
              "E_next": 0.0,
              "E_nn": 7.057910885364062,
              "E_nuc": 7.057910885364062,
              "E_tot": -460.6808100558237,
              "E_x": -6.968215269343292,
              "E_xc": -21.47954097593226,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -460.6808100558237,
            "energy_update": 9.908273852943239e-06,
            "mo_residual": 0.0011632387845317226,
            "wall_time": 60.30251131
          },
          {
            "energy_terms": {
              "E_ee": 210.75344314646108,
              "E_eext": 0.0,
              "E_el": -467.7387245297898,
              "E_en": -1110.1289909401464,
              "E_kin": 460.08414425470556,
              "E_next": 0.0,
              "E_nn": 7.057910885364062,
              "E_nuc": 7.057910885364062,
              "E_tot": -460.68081364442577,
              "E_x": -6.968114777008624,
              "E_xc": -21.47920621380139,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -460.68081364442577,
            "energy_update": 3.5886020555153664e-06,
            "mo_residual": 0.0008124797143635206,
            "wall_time": 59.567357521
          },
          {
            "energy_terms": {
              "E_ee": 210.75264346167728,
              "E_eext": 0.0,
              "E_el": -467.73872512824903,
              "E_en": -1110.1318168756025,
              "E_kin": 460.08779500556363,
              "E_next": 0.0,
              "E_nn": 7.057910885364062,
              "E_nuc": 7.057910885364062,
              "E_tot": -460.680814242885,
              "E_x": -6.96812857412648,
              "E_xc": -21.479218145761003,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -460.680814242885,
            "energy_update": 5.984592235108721e-07,
            "mo_residual": 0.0005379782600004602,
            "wall_time": 58.252142079
          },
          {
            "energy_terms": {
              "E_ee": 210.753292408509,
              "E_eext": 0.0,
              "E_el": -467.7387255754687,
              "E_en": -1110.1396382472017,
              "E_kin": 460.09511120181713,
              "E_next": 0.0,
              "E_nn": 7.057910885364062,
              "E_nuc": 7.057910885364062,
              "E_tot": -460.6808146901047,
              "E_x": -6.968161803903681,
              "E_xc": -21.479329134689515,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -460.6808146901047,
            "energy_update": 4.4721969061356504e-07,
            "mo_residual": 0.0005151860523357626,
            "wall_time": 61.257013759
          },
          {
            "energy_terms": {
              "E_ee": 210.75370410048865,
              "E_eext": 0.0,
              "E_el": -467.73872567973575,
              "E_en": -1110.1393757876695,
              "E_kin": 460.0944565513044,
              "E_next": 0.0,
              "E_nn": 7.057910885364062,
              "E_nuc": 7.057910885364062,
              "E_tot": -460.6808147943717,
              "E_x": -6.968165507441578,
              "E_xc": -21.479345036417733,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -460.6808147943717,
            "energy_update": 1.0426703056509723e-07,
            "mo_residual": 0.00012591610209040295,
            "wall_time": 58.846544899
          },
          {
            "energy_terms": {
              "E_ee": 210.75398123404605,
              "E_eext": 0.0,
              "E_el": -467.7387257179756,
              "E_en": -1110.138766498087,
              "E_kin": 460.09358037220875,
              "E_next": 0.0,
              "E_nn": 7.057910885364062,
              "E_nuc": 7.057910885364062,
              "E_tot": -460.6808148326116,
              "E_x": -6.96816714547436,
              "E_xc": -21.479353680668982,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -460.6808148326116,
            "energy_update": 3.823987526629935e-08,
            "mo_residual": 7.612589602359254e-05,
            "wall_time": 67.663087399
          }
        ],
        "wall_time": 797.866106876
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
