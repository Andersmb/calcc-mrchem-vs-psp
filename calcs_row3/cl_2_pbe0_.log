Starting job 5688112 on c1-[21,26],c2-7,c3-[6,13],c5-7 at Thu May 19 22:39:08 CEST 2022

Copying cl_2_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying cl_2_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688112      cl_2_pbe0_        180                                             00:15:38      0:0 
5688112.bat+      batch         24        1   00:00:02          0   00:00:02   00:15:38      0:0 
5688112.ext+     extern        180        6   00:00:00          5   00:00:00   00:15:38      0:0 
5688112.0      mrchem.x        180       30   00:14:24         29   00:33:48   00:14:32      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688112                                                                          
5688112.bat+     19332K          0     19332K        0              0          0 
5688112.ext+          0          5          0        0              5          0 
5688112.0      5302264K         29   2469448K        0             16          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688112                                                                                               
5688112.bat+       19.41M               0         19.41M        0.06M                0          0.06M 
5688112.ext+        0.00M               5          0.00M            0                5              0 
5688112.0           6.75M               0          6.73M       59.21M                0         32.26M 

Job 5688112 completed at Thu May 19 22:54:47 CEST 2022
