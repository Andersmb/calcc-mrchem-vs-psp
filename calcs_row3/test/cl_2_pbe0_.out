                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Cl            :        0.000000        0.000000       -0.066586
    1       H            :        0.000000        0.000000        2.342059
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               9
 Beta electrons          :                                               9
 Total electrons         :                                              18
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              1.000000000046e+00
    1    1    a          :                              1.000000000034e+00
    2    1    a          :                              9.999999999755e-01
    3    1    a          :                              9.999999999473e-01
    4    1    a          :                              1.000000000000e+00
    5    1    a          :                              9.999999998306e-01
    6    1    a          :                              9.999999993380e-01
    7    1    a          :                              9.999999999859e-01
    8    1    a          :                              1.000000000000e+00
    9    1    b          :                              1.000000000046e+00
   10    1    b          :                              1.000000000034e+00
   11    1    b          :                              9.999999999755e-01
   12    1    b          :                              9.999999999473e-01
   13    1    b          :                              1.000000000000e+00
   14    1    b          :                              9.999999998306e-01
   15    1    b          :                              9.999999993380e-01
   16    1    b          :                              9.999999999961e-01
   17    1    b          :                              1.000000000000e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                457.767552792617
 E-N energy              :            (au)              -1106.333062876738
 Coulomb energy          :            (au)                211.561127835617
 Exchange energy         :            (au)                 -6.975442045055
 X-C energy              :            (au)                -21.482414926277
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  7.057910885364
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -465.462239219837
 Nuclear energy          :            (au)                  7.057910885364
---------------------------------------------------------------------------
 Total energy            :            (au)             -4.584043283345e+02
                         :      (kcal/mol)             -2.876530566605e+05
                         :        (kJ/mol)             -1.203540389849e+06
                         :            (eV)             -1.247381668690e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)               -100.907968280360
    1    1    a          :            (au)                 -9.386454657273
    2    1    a          :            (au)                 -7.130062050990
    3    1    a          :            (au)                 -7.119101873107
    4    1    a          :            (au)                 -7.118670658372
    5    1    a          :            (au)                 -0.854422700688
    6    1    a          :            (au)                 -0.454206339968
    7    1    a          :            (au)                 -0.309670943753
    8    1    a          :            (au)                 -0.309113921412
    9    1    b          :            (au)               -100.907968280345
   10    1    b          :            (au)                 -9.386454657297
   11    1    b          :            (au)                 -7.130062051034
   12    1    b          :            (au)                 -7.119101873114
   13    1    b          :            (au)                 -7.118670658389
   14    1    b          :            (au)                 -0.854422701014
   15    1    b          :            (au)                 -0.454206342250
   16    1    b          :            (au)                 -0.309670944319
   17    1    b          :            (au)                 -0.309113921994
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -267.179342855678
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          4.242641e+00        -458.404328334473         -4.584043e+02
    1          1.822760e-01        -460.602195737053         -2.197867e+00
    2          6.339440e-02        -460.667962340451         -6.576660e-02
    3          2.524697e-02        -460.677281850841         -9.319510e-03
    4          1.587403e-02        -460.679542977766         -2.261127e-03
    5          1.554666e-02        -460.680586579436         -1.043602e-03
    6          6.732048e-03        -460.680740666488         -1.540871e-04
    7          4.785242e-03        -460.680800147550         -5.948106e-05
    8          1.163239e-03        -460.680810055824         -9.908274e-06
    9          8.124797e-04        -460.680813644426         -3.588602e-06
   10          5.379783e-04        -460.680814242885         -5.984592e-07
   11          5.151861e-04        -460.680814690105         -4.472197e-07
   12          1.259161e-04        -460.680814794372         -1.042670e-07
   13          7.612590e-05        -460.680814832612         -3.823988e-08
---------------------------------------------------------------------------
                      SCF converged in 13 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Cl            :        0.000000        0.000000       -0.066586
    1       H            :        0.000000        0.000000        2.342059
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                460.093580372209
 E-N energy              :            (au)              -1110.138766498087
 Coulomb energy          :            (au)                210.753981234046
 Exchange energy         :            (au)                 -6.968167145474
 X-C energy              :            (au)                -21.479353680669
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  7.057910885364
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -467.738725717976
 Nuclear energy          :            (au)                  7.057910885364
---------------------------------------------------------------------------
 Total energy            :            (au)             -4.606808148326e+02
                         :      (kcal/mol)             -2.890815734941e+05
                         :        (kJ/mol)             -1.209517304284e+06
                         :            (eV)             -1.253576303756e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)               -101.715136464959
    1    1    a          :            (au)                 -9.541618128593
    2    1    a          :            (au)                 -7.275514408554
    3    1    a          :            (au)                 -7.267725769942
    4    1    a          :            (au)                 -7.267725632167
    5    1    a          :            (au)                 -0.870235141385
    6    1    a          :            (au)                 -0.489498564290
    7    1    a          :            (au)                 -0.349309667071
    8    1    a          :            (au)                 -0.349309511130
    9    1    b          :            (au)               -101.715136465255
   10    1    b          :            (au)                 -9.541618128912
   11    1    b          :            (au)                 -7.275514409679
   12    1    b          :            (au)                 -7.267725770132
   13    1    b          :            (au)                 -7.267725632353
   14    1    b          :            (au)                 -0.870235138839
   15    1    b          :            (au)                 -0.489498552099
   16    1    b          :            (au)                 -0.349309667783
   17    1    b          :            (au)                 -0.349309511839
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -270.252146564982
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000000        0.000003       -0.773156
 Magnitude               :            (au)                        0.773156
                         :         (Debye)                        1.965167
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        1.210096
 Magnitude               :            (au)                        1.210096
                         :         (Debye)                        3.075757
---------------------------------------------------------------------------
 Total vector            :        0.000000        0.000003        0.436940
 Magnitude               :            (au)                        0.436940
                         :         (Debye)                        1.110590
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 14m 25s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
